From fb158dfa719e9fbc75e21b850adbe0728e80b120 Mon Sep 17 00:00:00 2001
From: DasBlub <dasblub@gmail.com>
Date: Fri, 26 Feb 2010 17:17:47 +0100
Subject: [PATCH] Add newline at end of file (make linux happy :))

---
 Tools/qemu/.gitignore |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Tools/qemu/.gitignore b/Tools/qemu/.gitignore
index 6fce5ec..f2ea08a 100644
--- a/Tools/qemu/.gitignore
+++ b/Tools/qemu/.gitignore
@@ -1 +1,2 @@
-stderr.txt
\ No newline at end of file
+stderr.txt
+
-- 
1.6.3.3

