Index: Prebuild/src/Core/Targets/NAntTarget.cs
===================================================================
--- Prebuild/src/Core/Targets/NAntTarget.cs	(revision 277)
+++ Prebuild/src/Core/Targets/NAntTarget.cs	(working copy)
@@ -297,6 +297,11 @@
                     ss.Write(" define=\"{0}\"", conf.Options.CompilerDefines);
                     break;
                 }
+				foreach (ConfigurationNode conf in project.Configurations) {
+					ss.Write(" nostdlib=\"{0}\"", conf.Options["NoStdLib"]);
+					break;
+				}
+
                 ss.Write(" main=\"{0}\"", project.StartupObject);
 
                 foreach (ConfigurationNode conf in project.Configurations)
@@ -498,6 +503,16 @@
                 ss.WriteLine("    </target>");
                 ss.WriteLine();
 
+				ss.WriteLine("    <target name=\"net-3.5\" description=\"Sets framework to .NET 3.5\">");
+				ss.WriteLine("        <property name=\"nant.settings.currentframework\" value=\"net-3.5\" />");
+				ss.WriteLine("    </target>");
+				ss.WriteLine();
+
+				ss.WriteLine("    <target name=\"mono-3.5\" description=\"Sets framework to mono 3.5\">");
+				ss.WriteLine("        <property name=\"nant.settings.currentframework\" value=\"mono-3.5\" />");
+				ss.WriteLine("    </target>");
+				ss.WriteLine();
+
                 ss.WriteLine("    <target name=\"mono-2.0\" description=\"Sets framework to mono 2.0\">");
                 ss.WriteLine("        <property name=\"nant.settings.currentframework\" value=\"mono-2.0\" />");
                 ss.WriteLine("    </target>");
Index: Prebuild/src/Core/Targets/VSGenericTarget.cs
===================================================================
--- Prebuild/src/Core/Targets/VSGenericTarget.cs	(revision 277)
+++ Prebuild/src/Core/Targets/VSGenericTarget.cs	(working copy)
@@ -270,6 +270,7 @@
 					ps.WriteLine("    <RemoveIntegerChecks>{0}</RemoveIntegerChecks>", conf.Options["RemoveIntegerChecks"]);
 					ps.WriteLine("    <TreatWarningsAsErrors>{0}</TreatWarningsAsErrors>", conf.Options["WarningsAsErrors"]);
 					ps.WriteLine("    <WarningLevel>{0}</WarningLevel>", conf.Options["WarningLevel"]);
+					ps.WriteLine("    <NoStdLib>{0}</NoStdLib>", conf.Options["NoStdLib"]); 
 					ps.WriteLine("    <NoWarn>{0}</NoWarn>", conf.Options["SuppressWarnings"]);
 					ps.WriteLine("  </PropertyGroup>");
 				}
@@ -680,7 +681,8 @@
 
 		private void WriteProject(SolutionNode solution, StreamWriter ss, DatabaseProjectNode dbProject)
 		{
-			WriteProject(ss, solution, "Database", dbProject.Guid, dbProject.Name, dbProject.FullPath);
+			if (null != solution.Files && solution.Files.Count > 0)
+				WriteProject(ss, solution, "Database", dbProject.Guid, dbProject.Name, dbProject.FullPath);
 		}
 
 		const string ProjectDeclarationBeginFormat = "Project(\"{0}\") = \"{1}\", \"{2}\", \"{3}\"";
